tests/suites/test_suite_ssl.function

```diff
@@ -12,6 +12,11 @@
 
 #include <test/constant_flow.h>
 
+#define MBEDTLS_SSL_TLS1_3_LABEL( idx, name, string )   \
+    const int tls1_3_label_ ## name = idx;
```

Comment:
I wonder if we could get rid of `idx` by using an enum here instead. If I'm reading [6.7.2.2 par. 3 of the standard](http://www.open-std.org/jtc1/sc22/wg14/www/docs/n1256.pdf#page=117) correctly, values are guaranteed to start from 0 and go up by increments of 1, which is exactly what we want here. I think it would make the list easier to maintain, by removing the need to manually update index values (as demonstrated by subsequent commits). Or am I missing something that would prevent this from working or make it fragile?

Reply:
Yes, the value of an enum constant declared without an explicit value is one plus the previous value, and it's 0 for the first constant if it doesn't have a value. I agree that it would be better not to have `idx` here, since having `idx` introduces the risk of duplicates.

Reply:
Much better - I'll follow your suggestion.

Reply:
Fixed.